### DISCUSSION TITLE
Notification link missing module link

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14477] [How to]
notification shows link rather then title linked to the record...  ideally ti should show a hyperlink to the record... 

please see this screenshot from gitdeveloper.. i have tested same on gitstable too...

![image](https://user-images.githubusercontent.com/54880467/112764436-204ce780-9026-11eb-8896-825882665d95.png)

any help is greatly appreciated sir..

**Comment by mariuszkrzaczkowski** [comment]
show your templates for sending notifications

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski  pls check :
![gdn](https://user-images.githubusercontent.com/54880467/112808151-52962d80-9096-11eb-8d93-bd56b9625ccb.gif)

it doesnt show module name and link to that record.. which notifications show for default in other modules like contacts, sale order etc,,
![image](https://user-images.githubusercontent.com/54880467/112808861-0e575d00-9097-11eb-9ee6-e1e6c21be28d.png)

 ..

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski pls check if u get time buddy...

**Reply by rioshaz** [reply]
community suport is very limited here i see...

**Comment by Haaaappi** [comment]
has anyone found a solution?

**Comment by mariuszkrzaczkowski** [comment]
there have been some improvements recently, check gitdeveloper
https://github.com/YetiForceCompany/YetiForceCRM/search?o=desc&q=Notification&s=committer-date&type=commits https://github.com/YetiForceCompany/YetiForceCRM/commit/58f5809f291220f47c6e3c5ea3fc3f0ffa38b301
